sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,16 +20,63 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
     dataUrl:
-      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
       configurableParameters:
         pageSizeParameterName: per-page
         defaultPageSize: 100
         pageParameterName: page
     response:
       itemsKeyFromResponseRoot:
         - items
+  # civi export
+  - dataset: '{ENV}'
+    table: civi_data_export
+    stateFile:
+      bucketName: '{ENV}-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-civi-export-processing-state.json'
+    urlSourceType:
+      name: 'civi'
+      typeSpecificVal:
```

Comment:
minor: `sourceTypeSpecificValues` might be clearer (otherwise by `type` I would think of basic types such as `string`)

Reply:
`urlSourceType` instead of `type`?